docs/lsm/embedded_entities/embedded_entities.rst

```diff
@@ -236,38 +236,103 @@ For these more involved update scenarios we recommend updating the lifecycle spe
+  state                 validating   current attributes   previous attributes   is like             operation since is like  
+====================== ============ ==================== ===================== =================== ========================= 
+  up                                 active               -                     -                   -                        
+  update_start                                                                  up                  -                        
+  update_start          yes                                                     update_inprogress   promote/backwards        
+  update_rejected                                                               up                  -                        
+  update_acknowledged                                                           up                  -                        
+  update_inprogress                  active               rollback              -                   -                        
+  rollback                           active               candidate             -                   -                        
+====================== ============ ==================== ===================== =================== ========================= 
+
+5. copy over the state of the is_like and apply the operations
+
+====================== ============ ==================== ===================== =================== ========================= 
+  state                 validating   current attributes   previous attributes   is like             operation since is like  
+====================== ============ ==================== ===================== =================== ========================= 
+  up                                 active               -                     -                   -                        
+  update_start                       active               -                     up                  -                        
+  update_start          yes          candidate            active                update_inprogress   promote/backwards        
+  update_rejected                    active               -                     up                  -                        
+  update_acknowledged                active               -                     up                  -                        
+  update_inprogress                  active               rollback              -                   -                        
+  rollback                           active               candidate             -                   -                        
+====================== ============ ==================== ===================== =================== ========================= 
+
+
+6. Finally, translate to the state variables as follows:
+
+   1. on all non-validating state, current attributes should be ``active``
```

Comment:
This is not something that can be set in the State, right ? Is this more something to check the integrity of this table before using it to define the states ?
I don't understand the added value of this line.

Reply:
It is to check you have something that makes sense and that is also  its value. 

```suggestion
   1. on all non-validating state, current attributes should be ``active``. If it is not, either you made a mistake or the state tracking plugin can't be used for this feature and you will have to build one yourself. 
```